aws_xray_sdk/core/plugins/ec2_plugin.py

```diff
@@ -1,7 +1,10 @@
 import json
 import logging
-from future.standard_library import install_aliases
-install_aliases()
+try:
+    from future.standard_library import install_aliases
+    install_aliases()
+except ImportError:
+    pass
```

Comment:
Instead of using `try-except`, can we reuse the [`PY2`](https://github.com/aws/aws-xray-sdk-python/blob/d025f5641496efdc9dfad2b270139c224053885c/aws_xray_sdk/core/utils/compat.py#L5) variable from `compat.py`?

Then this could be something like:
```python
from aws_xray_sdk.core.utils.compat import PY2
    
if PY2:
    from future.standard_library import install_aliases
    install_aliases()
```

Reply:
Oh interesting! I took this path because [StackOverflow says it was the status quo](https://stackoverflow.com/a/3496790/7460739) and I saw it [used in OpenTelemetry Python Contrib](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/0007c9046c74d724c342c769bca2a874357be907/instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware.py#L46-L52), but this `PY2` looks super useful so I'll change it to that :)

Reply:
I see. That's good to know. :)
Since we already have the other mechanism, I think it would be better to just reuse for the sake of consistency.